newdle/client/src/selectors.js

```diff
@@ -96,10 +96,18 @@ export const newdleHasParticipantsWithEmail = createSelector(
   getNewdleParticipants,
   participants => participants.some(p => p.email !== null)
 );
+export const newdleParticipantsWithEmail = createSelector(
```

Comment:
What do you think about composing the selectors to avoid repeating the condition `p => p.email !== null`?

Such as:
```
export const newdleParticipantsWithEmail = createSelector(
  getNewdleParticipants,
  participants => participants.filter(p => p.email !== null)
);
export const newdleHasParticipantsWithEmail = createSelector(
  newdleParticipantsWithEmail,
  participants => participants.length
);
```

Reply:
I was actually thinking about commenting the exact same thing, but then decided against it because in the end it's very small and - not that it matters even for large newdles - `some()` is still more efficient since it stops at the first successful check.

Anyway, if you decide to change it, make sure to convert to bool, i.e. `!!participants.length` or `participants.length !== 0`

Reply:
Good spot, changed